### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by hellonadya**
- change names in acc. with 6.0
- change comparison table
- add info on oo workspace

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -34,44 +40,46 @@ The editors allow you to create, edit, save and export text, spreadsheet and pre
 
 ONLYOFFICE offers different versions of its online document editors that can be deployed on your own servers.
 
-ONLYOFFICE Document Server:
+ONLYOFFICE Docs:
 
 * Community Edition (`onlyoffice-documentserver` package)
-* Integration Edition (`onlyoffice-documentserver-ie` package)
+* Enterprise Edition (`onlyoffice-documentserver-ee` package)
```
Comment (ShockwaveNN) [inline Readme.md:46]: Somewhere in this instruction should be info about

 `I'm currently using documentser-ie. What should I do to use documentserver-ee`

File: Readme.md [hunk 2]
```diff
@@ -34,44 +40,46 @@ The editors allow you to create, edit, save and export text, spreadsheet and pre
 
 ONLYOFFICE offers different versions of its online document editors that can be deployed on your own servers.
 
-ONLYOFFICE Document Server:
+ONLYOFFICE Docs:
```
Comment (ShockwaveNN) [inline Readme.md:43]: Add info, that name change, but packages names are still same

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 2 above)
Comment (hellonadya) [reply Readme.md]: `ONLYOFFICE Docs (packaged as Document Server):`

is that OK?

File: Readme.md (hunk 1 above)
Comment (hellonadya) [reply Readme.md:46]: Agreed. @papacarlo what do you think?

File: Readme.md (hunk 2 above)
Comment (agolybev) [reply Readme.md]: I agree.

File: Readme.md (hunk 1 above)
Comment (agolybev) [reply Readme.md:46]: You have to install documentserver-ee

**Pull request merged**